### PR TITLE
Update networks.json - Add Candle

### PR DESCRIPTION
**Pull request by samisbakedham**
Adds support for Candle chain on snapshot.

Changes proposed in this pull request:
- addition of a new network (candle)  
- 
-

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -710,6 +710,21 @@
     "start": 2762929,
     "logo": "ipfs://QmXLZyAr6UNFQ4tkNwSyeNByFvzwYpwiNgV5vHuoxn74Rg"
   },
+  "534": {
+    "key": "534",
+    "name": "Candle",
+    "shortName": "CNDL",
+    "chainId": 534,
+    "network": "mainnet",
+    "multicall": "0x7839444C21C03F03cF7CD29718703D5e9B0B010C",
```
Comment (ChaituVR) [inline src/networks.json:719]: Hey, @samisbakedham Looks good, except that the multicall code here is not verified, can you verify and publish the code to explorer plz. :) Thanks

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (samisbakedham) [reply src/networks.json:719]: Fixed!

**Comment by samisbakedham** [comment]
Fixed

**Comment by samisbakedham** [comment]
> @samisbakedham What is the exact URL, when i try to go here https://candleexplorer.com/address/0x7839444C21C03F03cF7CD29718703D5e9B0B010C/contracts I can see it is still not verified?

You're right, that contract had yet to be verified on the explorer. Have now deployed the new Multicall contract and verified it on the explorer. Updated the address in the PR as well. [Here](https://candleexplorer.com/address/0x4A21871491adC2C429F9903918C306c97dd295A3/contracts) is the link to the verified contract on Candle Explorer. [Here ](https://repo.sourcify.dev/contracts/full_match/534/0x4A21871491adC2C429F9903918C306c97dd295A3/)is also the link to the verified contract on Sourcify's server for Candle. Thank you!

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**